packages/components/src/widgets/ArrayWidget.tsx

```diff
@@ -3,18 +3,34 @@ import { useMemo } from "react";
 import { ValueViewer } from "../components/SquiggleViewer/ValueViewer.js";
 import { widgetRegistry } from "./registry.js";
 import { SqTypeWithCount } from "./SqTypeWithCount.js";
+import { SqValueWithContext } from "../lib/utility.js";
+import { DocumentTextIcon } from "@quri/ui";
+
+function isNotebook(value: SqValueWithContext) {
+  return Boolean(value.tags.notebook());
+}
 
 widgetRegistry.register("Array", {
   heading: (value) => `List(${value.value.getValues().length})`,
-  Preview: (value) => (
-    <SqTypeWithCount type="[]" count={value.value.getValues().length} />
-  ),
+
+  Preview: (value) => {
+    return isNotebook(value) ? (
+      <DocumentTextIcon size={14} className="opacity-40" />
```

Comment:
Idea: add a tooltip with "Notebook with N cells"

Reply:
We really don't make the cells very clear now, I think this could confuse more than help. One "cell" could be markdown of a description for a previous item, plus a header for the next one, which is awkward. And there's little way to see a distinction.